tiledb/CMakeLists.txt

```diff
@@ -617,6 +617,20 @@ if(TILEDB_WEBP)
   )
 endif()
 
+if(TILEDB_ABSEIL)
+  find_package(absl_EP REQUIRED)
+  find_package(absl CONFIG REQUIRED
```

Comment:
Why do we have the second `find_package(absl` call? The `absl_EP` should take care of finding the package and making the targets available. This is how the other EP packages work.

Reply:
removed